domain/test/fixture/test_fixture.dart

```diff
@@ -90,4 +90,9 @@ final accountQuota1 = AccountQuota(
   usedSpace: QuotaSize(20),
   maxFileSize: QuotaSize(30),
   maintenance: true
-);
\ No newline at end of file
+);
+
+final Sorter sharedSpaceSorted = Sorter(OrderScreen.sharedSpaceWorkGroup, OrderBy.modificationDate, OrderType.descending);
+final Sorter sharedSpaceShared = Sorter(OrderScreen.sharedSpaceWorkGroup, OrderBy.shared, OrderType.descending);
+final Sorter sharedSpaceSender = Sorter(OrderScreen.sharedSpaceWorkGroup, OrderBy.sender, OrderType.descending);
```

Comment:
You should rename these in "sharedSpaceSenderSorter" so we know it's a sorter because it will be hard to use otherwise :(

Reply:
What you did there : https://github.com/linagora/linshare-mobile-flutter-app/pull/362/files#diff-9559f78892b0a38dba8ef612d3b1884de60dc93165f645404d4d88f3c3920addR166

Reply:
It's used for `sort_interactor_test.dart`